package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "api-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Hey Daisy! This line is causing the tests not to run! If you could switch the line to the following that would be great:
```
"test": "jest --verbose --coverage",
"test-watch": "jest --watchAll --verbose --coverage"
```